src/components/security_manager/src/security_manager_impl.cc

```diff
@@ -143,7 +143,7 @@ security_manager::SSLContext* SecurityManagerImpl::CreateSSLContext(
 
   security_manager::SSLContext* ssl_context = session_observer_->GetSSLContext(
       connection_key, protocol_handler::kControl);
-  // return exists SSLCOntext for current connection/session
+  // return existed SSLCOntext for current connection/session
```

Comment:
should be // return existing SLContext for current connection/session

Reply:
@mrapitis thanks, fixed in https://github.com/smartdevicelink/sdl_core/pull/2068/commits/f52af91122b5daf719747027a9fa248419dc6850